README.md

```diff
@@ -9,8 +9,9 @@ More Details : https://aternos.org/:en/
 2. Install the required libraries using `pip install -r requirements.txt`.
 3. Create a Discord bot and get the bot token from the Discord Developer Portal.
 4. Enable messages in privileged intents for your bot.
-5. Fill in your Aternos credentials in the env file in the ATERNOS_USERNAME and ATERNOS_PASSWORD fields
-6. Start the bot using `python bot.py`.
+5. Rename the .env.sample file to `.env`
```

Comment:
Dont say rename. If rename means they need to commit it. Say copy the sample file and paste as .env to do modification.
This way, renamed file / adding new .env will not be part of their git status and commit.